### ISSUE TITLE
MacOSX working with 2x scaled resolution but not 1x

### ISSUE THREAD

**Issue by Kiruscha** [Detergent13/tft-bot#73]
Hello,

trying to run this on following device:
M1 MacBook Air.

Issue: 
When using default Mac resolution of 2x scaled, for example 1680x1050 (2x) this script can start up queue just fine.

However, when using 1x non-scaled resolution of 2560x1600 (1x), then this script does not start up queue.

Yes, every other setting (obviously) is still the same.

What could this be? How could I fix this? 

Sidenote: I am running this in 2560x1600 so that league can run in 1920x1080 without me having to recapture everything. Yes, that would be extremely painful, especially because some screens cannot be recaptured playing a single game (mission dialog etc).

Going to patiently wait for an answer. :) 
Biggest help would be if someone else with an M1 MacBook running this script could tell me what they did.

**Comment by Kiruscha** [comment]
fix: implement python-imagesearch instead of python-imagesearch-drov0 (that is the updated version and it includes fixes for retina displays, which Apple uses).

**Comment by Detergent13** [comment]
Good catch! I didn't realise that dependency was deprecated. If you're currently using this new library, what version seems to be working for you?
